### PR TITLE
Component structure 

### PR DESCRIPTION
**Pull request by learnwithalfred**
Components has been setup

- All the boilerplate from CRA including text, images, styles deleted
- Directory called components created
- Minimal styling added to calculator

Screenshot 2022-06-27 at 10 32 24

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (DammyShittu) [inline src/App.js]: - [x] According to the requirements all components should be class-based. Kindly refactor this component accordingly.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @learnwithalfred,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] Kindly delete the following boilerplate files that won't likely won't be needed for this project.

  public directory | src directory
  :---:|:---:
  alfred-deletes | alfred-deletes-2

- [x] In your README kindly include details on how to clone and run the app locally. ie:
  - git clone https://github.com/learnwithalfred/alfred-maths-magician-app.git
  - cd alfred-maths-magician-app
  - npm install
  - npm start

  alfred-readme

- [x] _Check comments under review_


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (learnwithalfred) [reply src/App.js]: You are right 

Thank you. Let me make the updates

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @learnwithalfred, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Good commit messages 👍 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by learnwithalfred** [comment]
Please just one note.

I had started working on the next requirement. So the Logic folder was coincidentally committed with this changes.

But I hope it will not cause any issues

**Pull request merged**